Issue with node 6.0.0

I tried out `npm run build` with node version 6.0.0 (npm version 3.8.7, I believe) and it produced the error below. I haven't had a chance to debug it, yet - sorry!

```
> docbox@1.0.0 build /path/to/docbox
> NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js && npm run prerender -- index.html

TypeError: Path must be a string. Received undefined while parsing file: /path/to/docbox/src/index.js
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at /path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:353:36
    at /path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:375:22
    at Array.map (native)
    at OptionManager.resolvePresets (/path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:364:20)
    at OptionManager.mergePresets (/path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:348:10)
    at OptionManager.mergeOptions (/path/to/docbox/node_modules/babel-core/l
> docbox@1.0.0 prerender /path/to/docbox
> babel src --out-dir lib && node lib/render.js "index.html"

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at /path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:353:36
    at /path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:375:22
    at Array.map (native)
    at OptionManager.resolvePresets (/path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:364:20)
    at OptionManager.mergePresets (/path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:348:10)
    at OptionManager.mergeOptions (/path/to/docbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:307:14)
    at /Users/me/Documents/
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/me/n/bin/node" "/Users/me/n/bin/npm" "run" "prerender" "--" "index.html"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.7
npm ERR! code ELIFECYCLE
npm ERR! docbox@1.0.0 prerender: `babel src --out-dir lib && node lib/render.js "index.html"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the docbox@1.0.0 prerender script 'babel src --out-dir lib && node lib/render.js "index.html"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the docbox package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     babel src --out-dir lib && node lib/render.js "index.html"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs docbox
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls docbox
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /path/to/docbox/npm-debug.log

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/me/n/bin/node" "/Users/me/n/bin/npm" "run" "build"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.7
npm ERR! code ELIFECYCLE
npm ERR! docbox@1.0.0 build: `NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js && npm run prerender -- index.html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the docbox@1.0.0 build script 'NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js && npm run prerender -- index.html'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the docbox package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js && npm run prerender -- index.html
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs docbox
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls docbox
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /path/to/docbox/npm-debug.log
```

--- a/circle.yml
+++ b/circle.yml
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 5
+    version: 6
 test:
   post:
     - npm run build

